pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -60,8 +60,13 @@ type LabeledCommand struct {
 }
 
 // +k8s:openapi-gen=true
-// +union
 type Command struct {
+	LabeledCommand `json:",inline"`
```

Comment:
I would not do it like that.
I would keep BaseCommand and LabeledCommand where they are, and would bring only the ID to the top level.

Reply:
I noticed that vscode commands don't have `label`, is that intentional?

Reply:
yes, because the label is expected to be in the VSCode raw json definition afair.

Reply:
ok makes sense